third_party/crashpad/crashpad/util/net/http_transport_socket.cc

```diff
@@ -329,7 +329,11 @@ base::ScopedFD CreateSocket(const std::string& hostname,
         } else if (WaitUntilSocketIsReady(result.get())) {
           return result;
         }
-        return base::ScopedFD();
+        #if defined(STARBOARD) || defined(NATIVE_TARGET_BUILD)
+            continue;
+        #else
```

Comment:
Thanks for using the preprocessor directives, but please keep them unindented. There should be a few examples in this file, for reference.